app/models/qualifications.rb

```diff
@@ -0,0 +1,27 @@
+class Qualifications
+  def initialize(profpost_flag:, is_pgde:, is_further_education:)
+    @profpost_flag = profpost_flag
+    @is_pgde = is_pgde
+    @is_further_education = is_further_education
+  end
+
+  def to_a
+    qts_or_not + uni_qualification_or_not
+  end
+
+private
+
+  def qts_or_not
+    @is_further_education ? [] : [:qts]
+  end
+
+  def uni_qualification_or_not
```

Comment:
Maybe a comment or two with regards to `uni_qualification ` vs `further_education`

As layman terms,  uni and further education are synonymous to one and another in most sense.

Is uni_qualification just another way of stating it will be uni accreditation of a qualification?

Reply:
I've renamed the methods for clarity – can you please have another look?

Reply:
much better but what it is awar`e`ded

Reply:
> much better but what it is awar`e`ded

fixed